server/repository/repository.go

```diff
@@ -85,6 +85,37 @@ func (s *Server) List(ctx context.Context, q *repositorypkg.RepoQuery) (*appsv1.
 	return s.ListRepositories(ctx, q)
 }
 
+// Get return the requested configured repository by URL and the state of its connections.
+func (s *Server) Get(ctx context.Context, q *repositorypkg.RepoQuery) (*appsv1.Repository, error) {
```

Comment:
I think the `GetRepository()` call should be within the RBAC enforcer condition, as to prevent guessing of repository URLs that are configured.

Reply:
Taken into account. Should be fine now.